### PR TITLE
[skip ci] Update README.md

### PR DESCRIPTION
**Pull request by jielinxu**

### CODE REVIEW

**Review by yamasite — CHANGES_REQUESTED** [review]
IVFLAT test report links are broken.

File: docs/README.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ The following is a list of existing test reports:
 
 - [IVF_SQ8](test_report/milvus_ivfsq8_test_report_detailed_version.md)
 - [IVF_SQ8H](test_report/milvus_ivfsq8h_test_report_detailed_version.md)
+- [IVFLAT](test-report/ivfflat_test_report_en.md)
 
 To read the CN version of these reports:
 
 - [IVF_SQ8_cn](test_report/milvus_ivfsq8_test_report_detailed_version_cn.md)
 - [IVF_SQ8H_cn](test_report/milvus_ivfsq8h_test_report_detailed_version_cn.md)
+- [IVFLAT](test-report/ivfflat_test_report_cn.md)
```
Comment (JinHai-CN) [inline docs/README.md]: why IVFLAT missing _cn?

Comment (yamasite) [inline docs/README.md]: Still missing _cn.

Comment (jielinxu) [reply docs/README.md]: Done

**Review by yamasite — APPROVED** [review]

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**